Sasquatch/Sasquatch/ViewControllers/Cells/MSCustomPropertyTableViewCell.swift

```diff
@@ -92,6 +96,18 @@ import UIKit
     prepareForReuse()
   }
   
+  override func didTransition(to state: UITableViewCell.StateMask) {
+    
+    // We need to pass the parent view controller to the picker
+    // to be able to show the alert on mac catalyst.
+    // We need to do it here, where we know the viewController is already set.
+#if targetEnvironment(macCatalyst)
```

Comment:
Why platform specific logic is placed out of `MSEnumPicker`? Can we put all conditionals inside? if you need extra data there it probably makes sense to just pass it every time